packages/devtools_app/integration_test/test/live_connection/debugger_panel_test.dart

```diff
@@ -147,6 +148,26 @@ void main() {
     // Verify that line 40 is focused:
     expect(isLineFocused(line40Finder), isTrue);
   });
+
+  test('invalid setBreakpoint throws exception', () async {
+    // Error codes defined by https://www.jsonrpc.org/specification#error_object
+    const jsonRpcInvalidParamsCode = -32602;
+
+    await expectLater(
+      serviceManager.service!.addBreakpoint(
+        serviceManager.isolateManager.selectedIsolate.value!.id!,
+        'fake-script-id',
+        1,
+      ),
+      throwsA(
+        const TypeMatcher<RPCError>().having(
+          (e) => e.code,
+          'code',
+          equals(jsonRpcInvalidParamsCode),
```

Comment:
If we update `vm_service` to `^11.7.1` we can use `RPCErrorKind.kInvalidParams.code`